core/src/main/java/org/geysermc/geyser/registry/populator/CreativeItemRegistryPopulator.java

```diff
@@ -49,8 +49,6 @@
 
 public class CreativeItemRegistryPopulator {
     private static final List<BiPredicate<String, Integer>> JAVA_ONLY_ITEM_FILTER = List.of(
-            // Just shows an empty texture; either way it doesn't exist in the creative menu on Java
-            (identifier, data) -> identifier.equals("minecraft:debug_stick"),
             // Bedrock-only as its own item
             (identifier, data) -> identifier.equals("minecraft:empty_map") && data == 2,
             // Bedrock-only banner patterns
```

Comment:
So these are predicates to ignore creative items that are exclusive to bedrock right? `BEDROCK_ONLY_ITEM_FILTER` ?

Reply:
yup - I don't know why the debug stick is even on here

Reply:
No texture seems like a fair reason to exclude :p

Reply:
If you look at the creative contents/item mappings for 1.20.80/1.21.0, you'll see that no such mapping exists.. nor has it existed for the versions before. Maybe it did at one point, but currently, this would never be used